images/php-fpm/00-lagoon-php.ini.tpl

```diff
@@ -8,6 +8,7 @@ display_startup_errors = ${PHP_DISPLAY_STARTUP_ERRORS:-Off}
 auto_prepend_file = ${PHP_AUTO_PREPEND_FILE:-none}
 auto_append_file = ${PHP_AUTO_APPEND_FILE:-none}
 error_reporting = ${PHP_ERROR_REPORTING:-E_ALL & ~E_DEPRECATED & ~E_STRICT}
+error_log=${PHP_ERROR_LOG:-/proc/self/fd/2}
```

Comment:
Hm, I have deployed the test image that was built by the CI from this PR to a Lagoon environment and in the CLI container I get

```
[foobar]test@cli-drupal:/app$ php -i | grep error_log
error_log => /proc/self/fd/2 => /proc/self/fd/2
```

but `admin/reports/status/php` says error_log is unset...